chatmessage/models.py

```diff
@@ -0,0 +1,16 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+# from client.models import Client
+# from professional.models import Professional
+
+
+class Chatmessage(models.Model):
+    message_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    # professional_id = models.ForeignKey(Professional, on_delete=models.CASCADE)
+    # client_id = models.ForeignKey(Client, on_delete=models.CASCADE)
+    date = models.DateTimeField(default=timezone.now)
+    message = models.TextField()
+
```

Comment:
Django will automatically call your table chatmessage_ Chatmessage so you should add these lines :class Meta: db_table = 'Chatmessage' so that your table will be called Chatmessage

Reply:
> Django will automatically call your table chatmessage_ Chatmessage so you should add these lines :class Meta: db_table = 'Chatmessage' so that your table will be called Chatmessage

@patrisiakaplun any replies for this?

Reply:
@idosinger98 @Yarboa done